Dockerfile

```diff
@@ -33,19 +33,20 @@ COPY ./nx.json nx.json
 COPY ./replace.build.mjs replace.build.mjs
 COPY ./tsconfig.base.json tsconfig.base.json
 
-RUN npm run build:production
+# Update build command for Nx 21
+RUN npx nx run-many --target=build --projects=api,client --configuration=production
```

Comment:
Is this still the same? In `package.json`, there are currently more commands that seem to be missing:
```
nx run api:copy-assets && nx run api:build:production && nx run client:copy-assets && nx run client:build:production && nx run ui:build-storybook && npm run replace-placeholders-in-build
```